### ISSUE TITLE
Multiple pizza events in future causes crash

### ISSUE THREAD

**Issue by thaliawww-service** [thaliawww/concrexit#606]
In GitLab by @se-bastiaan on Mar 12, 2018, 21:05

### One-sentence description

Multiple pizza events in future causes crash

### Current behaviour

When having multiple pizza events in the future the website crashes.

### Expected behaviour

No crash, duh.

### Steps to reproduce

1. Create two pizza events in le future.
2. Go to `/api/v1/pizzas/event/`
3.  

```
MultipleObjectsReturned at /api/v1/pizzas/event/
get() returned more than one PizzaEvent -- it returned 2!
```

**Comment by thaliawww-service** [comment]
In GitLab by @thomwiggers on Mar 20, 2018, 18:24

assigned to @thomwiggers

**Comment by thaliawww-service** [comment]
In GitLab by @thomwiggers on Mar 20, 2018, 18:29

mentioned in commit 938a986848a49ab70012af695aea0a0c022eb55c

**Comment by thaliawww-service** [comment]
In GitLab by @thomwiggers on Mar 21, 2018, 10:22

Actually, I guess I could write a test for this as well.

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Mar 22, 2018, 15:31

closed via merge request !769

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Mar 22, 2018, 15:31

closed via commit d5ca0efe7b289e411609b689f5a4eb559e05d0e8

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Mar 28, 2018, 19:48

mentioned in commit d5ca0efe7b289e411609b689f5a4eb559e05d0e8
